libmscore/point.h

```diff
@@ -0,0 +1,76 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2018 Werner Schweer
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __POINT_H__
+#define __POINT_H__
+
+#include "fraction.h"
+
+#include <climits>
+
+namespace Ms {
+
+class Element;
+class XmlReader;
+class XmlWriter;
+
+class PointInfo {
```

Comment:
@dmitrio95, maybe this class should be called `Move` or `UniveralMove` or something to make it clear that it corresponds to `<move>` in the MSCX?

Reply:
I support the idea on renaming both this class and the corresponding tag to something more consistent. For cross-staff chords I would rather return the old syntax (`<move>1</move>`) as it would be more consistent with other elements' properties which are normally written in separate tags which are not mixed with anything else. But this doesn't matter so much.

Concerning the name of the renamed tag, a `<pos>` tag denotes now an element offset in a score. Although I don't know reasons for that (there is an `<offset>` tag that denotes the same property) it can potentially be not too convenient to have both `<pos>` and `<position>` tags. So I think we either have to always use `<offset>` instead of `<pos>` or prefer `<location>` name for the renamed `<move>` tag.